source/02_algo/exploration_rl_en.rst

```diff
@@ -0,0 +1,241 @@
+
+    - Count-based intrinsic rewards
+    - Intrinsic reward based on prediction error
+    - Intrinsic rewards based on information theory
+
+3. Memory-based exploration
+
+    - Episodic Memory
+    - Direct exploration
+
+4. Other exploration mechanisms
+
+The representative algorithms of each research direction and their key points are shown in the following figure:
+
+.. image:: images/exploration_overview.png
+   :align: center
+   :scale: 25%
+
+Classic Exploration Mechanism
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In traditional multi-armed bandit problems, commonly used classical exploration mechanisms include:
+
+-  Epsilon-Greedy：At a certain moment, all actions have a choice probability greater than 0. The agent performs random actions (exploration) with a small probability\ :math:`\epsilon`\, and performs the action with the largest Q value with a large probability\ :math:`1-\epsilon`\ (using ).
+
+-  Upper Confidence
+   Bounds: The agent greedily chooses the action that maximizes the upper bound on the confidence \ :math:`\hat{Q}_{t}(a)+\hat{U}_{t}(a)`\, where \ :math:`\hat{Q}_{t}(a)`\ is in :math:`t` The average reward associated with the action:math:`a`\, \ :math:`\hat{U}_{t}(a)` is a reward associated with taking the action :math:`a`\ function that is inversely proportional to the number of times.
+
+-  Boltzmann Exploration: The agent samples actions from the Boltzmann distribution corresponding to the learned Q value (that is, the distribution obtained after performing the softmax operation on the logits of the Q value), and the degree of exploration can be adjusted by the temperature parameter.
+
+The following mechanisms can be used to get better exploration performance in deep reinforcement learning training when function approximation is performed by neural networks:
+
+-  Entropy
+   Loss：The agent is encouraged to perform more diverse actions by adding an additional entropy regularization term to the loss function.
+
+-  Noise-based
+   Exploration：Exploration is achieved by adding noise to the parameter space of observations, actions, and even the network.
+
+Intrinsic reward-based exploration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One of the more important methods in exploring mechanism design is to design special rewards to stimulate the "curiosity" of the agent. Generally, we refer to the rewards given by the environment as \**extrinsic rewards**\ , and the reward given by the exploration mechanism is called \**intrinsic reward**\.
+
+We hope to achieve two things by adding this additional intrinsic reward:
+
+1.  Exploration of the state space: motivating the agent to explore more novel states. (need to evaluate the novelty of state \ :math:`s`\  )
+
+2. Exploration of the state-action space: Incentivize the agent to perform actions that help reduce uncertainty about the environment. (Need to evaluate state-action pairs:math:`(s,a)` novelty)
+
+First give the definition of \**novelty**\ qualitatively:
```

Comment:
In general, please check "**" format. (I think maybe you are trying to make the text bold？）